sway/desktop/render.c

```diff
@@ -110,6 +139,11 @@ static void render_surface_iterator(struct sway_output *output, struct sway_view
 		return;
 	}
 
+	struct output_config *oc = find_output_config(output);
```

Comment:
How expensive is this call and Is it acceptable to call it in the iterator?  Should the ```scale_filter``` property be a member of ```sway_output```?

Reply:
I did think about that, but I don't know the answer.

Reply:
If you do that, you could move the calculation of the required filter from ```set_scale_filter``` to ```apply_output_config``` as well. That removes almost all of the new code from the rendering path.

Reply:
That's a great idea, I'll rework it to that.